homeassistant/components/notify/xmpp.py

```diff
@@ -14,8 +14,8 @@
 from homeassistant.const import (
     CONF_PASSWORD, CONF_SENDER, CONF_RECIPIENT, CONF_ROOM)
 
-REQUIREMENTS = ['sleekxmpp==1.3.2',
-                'dnspython3==1.15.0',
+REQUIREMENTS = ['slixmpp==1.4.0',
+                'aiodns==1.1.1',
                 'pyasn1==0.3.7',
                 'pyasn1-modules==0.1.5']
```

Comment:
`sleekxmpp` is taking care about it's [dependencies](https://github.com/poezio/slixmpp/blob/master/setup.py#L87).

Reply:
you mean `slixmpp` does. ok - I'll remove the extra dependencies.

Reply:
@fabaff what's the general rule here? packages that define their own dependencies will just get e.g. `slixmpp==1.4.0` defined, and no dependencies?
My tests and working setup are running with the explicitly defined versions from above. That could break with any update of `pyasn1`, `pyasn1-modules` or `aiodns` above 1.0.

I'm asking because `sleekxmpp` did [define dependencies as well](https://github.com/fritzy/SleekXMPP/blob/285495d5ee2427d93d961ceedcd1829383e5196d/setup.py#L156).

Reply:
Let's try to just use `slixmpp` as dependency. I think that I needed to add `dnspython3`, `pyasn1` and `pyasn1-modules` because they weren't pull in and later we were lucky to have them (https://github.com/fritzy/SleekXMPP/issues/477).

Reply:
ok.